components/brave_new_tab_ui/components/default/notification/index.ts

```diff
@@ -3,35 +3,34 @@
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 import styled from 'styled-components'
+import * as leo from '@brave/leo/tokens/css/variables'
```

Comment:
optional nit: not really important at all but I normally just import the bits I need from Nala

`import { color, font } from '@brave/leo/tokens/css/variables'`